onnxruntime/core/providers/cpu/nn/conv.cc

```diff
@@ -65,7 +67,7 @@ Status Conv<T>::Compute(OpKernelContext* context) const {
   AllocatorPtr alloc;
   ORT_RETURN_IF_ERROR(context->GetTempSpaceAllocator(&alloc));
 
-  auto col_data = alloc->Alloc(sizeof(T) * col_buffer_size);
+  auto col_data = alloc->Alloc(SafeInt<size_t>(sizeof(T)) * col_buffer_size);
```

Comment:
Shouldn't the SafeInt be around the whole expression? SafeInt on a sizeof(T) doesn't make sense.

Reply:
Ignore, I was thinking of some safe cast macro, this is apparently more modern now.